modules/capacity_factors_wind_pv/workflow/Snakefile

```diff
@@ -0,0 +1,50 @@
+import yaml
+
+from snakemake.utils import min_version, validate
+
+# Limit the snakemake version to a modern one.
+min_version("8.10")
+
+
+# Load the default configuration. This will be overridden by users.
+configfile: "config/default.yaml"
+
+
+# Validate the configuration using the schema file.
+# validate(config, workflow.source_path("schemas/config.schema.yaml"))
+
+# Load internal settings separately so users cannot modify them.
+with open(workflow.source_path("internal/internal_config.yaml"), "r") as f:
+    internal = yaml.safe_load(f)
+
+
+# Add all your includes here.
+include: "rules/downloads.smk"
+include: "rules/shapes.smk"
+include: "rules/wind-and-solar.smk"
+
+
+ALL_CF_TECHNOLOGIES = [
+    "wind-onshore",
+    "wind-offshore",
+    "open-field-pv",
+    "rooftop-pv",
+    "rooftop-pv-n",
+    "rooftop-pv-e-w",
+    "rooftop-pv-s-flat",
+]
+
+
+rule all:
+    message:
+        "Generate all outputs for 'capacity_factors_wind_pv'."
+    input:
+        expand(
+            "results/{resolution}/capacityfactors-wind-offshore.csv",
+            resolution=config["resolution"],
+        ),
+        expand(
+            "results/{resolution}/capacityfactors-{technology}.csv",
+            resolution=config["resolution"],
+            technology=ALL_CF_TECHNOLOGIES,
+        ),
```

Comment:
In the new IO, the all rule is deactivated via `use_default_user_config: False`.
If enabled, you should run a user input example.